core/types/transaction.go

```diff
@@ -46,6 +47,7 @@ const (
 	AccessListTxType
 	DynamicFeeTxType
 	StarknetType
+	BlobTxType = 5
```

Comment:
curious why specifically assign this `5` instead of use the default value, in this case `4`?

Reply:
not sure, it's from the spec though so it is correct. There may be another tx type in some other EIP that hasn't been implemented yet.